tests/test_imports.py

```diff
@@ -0,0 +1,93 @@
+"""Import tests."""
+
+import importlib
+import os
+import pkgutil
+import subprocess
+import sys
+from collections.abc import Sequence
+
+import pytest
+from pytest import param
```

Comment:
Is there any reason for this additional import and not just calling `pytest.param`?

Reply:
In this file it doesn't make a huge difference but I usually tend to import this separately to keep the parametrization code short. Have look, for instance, into `test_parameter_validation.py` and you'll see why 😄 

Since it pretty much comes at zero additional costs, there is no harm to also keep it here, so unless you really want me to remove it, I'd say let's just keep it for better readability